_posts/2018-09-13-september-office-hours.md

```diff
@@ -0,0 +1,87 @@
+---
+title: September Office Hours
+description: Multi-user OpenFaaS Cloud, Istio Service Mesh, and Stateless Microservices
+
+date: 2018-09-13
+image: /images/september-office-hours/team.jpg
+categories:
+  - office-hours
+  - community
+  - kubernetes
+author_staff_member: burton
+dark_background: true
+---
+
+This month's OpenFaaS office hours meeting talks about OpenFaaS Cloud as a multi-user platform for teams, how we've improved the ability to run stateless microservices managed as functions, and includes a demo of Istio service mesh integration.
+
+The OpenFaaS community holds regular office hours for anyone to join in as well as more focused discussions with project members
+
+Types of calls:
+* Office Hours (750 community members)
+  * Questions and Answers
+  * Project updates and direction
+* Members Call (15 members)
+  * Members of the Github organization
+  * Assign and track issues
+* Core Contributors (5 core members)
+  * Wide project vision
+  * Help with project direction
+
+> All of the office hours meetings are recorded and available for later viewing on YouTube.
+
+<iframe width="700" height="394" src="https://www.youtube.com/embed/IQCnHMn1wdI?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+Subscribe to the [Official OpenFaaS YouTube Channel](https://www.youtube.com/channel/UCdKi97g5FmzvrmtIp9FyOVA) where you can view the highlights from our meetings including demos of the newest features we're working on
+
+## Project Goals and Purpose
+OpenFaaS is focused on developers first, easy for operators, and the great community aspect
+
+* Developer first
+  * OpenFaaS gives you "Serverless on your terms"
+  * Focused on being easy to use with a web UI and CLI (eg: `faas-cli up`)
+    * [Five Tips and Tricks for the OpenFaaS CLI]({% post_url 2018-08-22-five-cli-tips %})
+* Easy for operators
+  * [OpenFaaS Operator](https://github.com/openfaas-incubator/openfaas-operator)
+  * [Scale to Zero]({% post_url 2018-07-25-zero-scale %})
+* Great Community
+  * 130+ contributors
+  * Most have multiple commits, coming back helping build the project
+  * Built by the community, for you to use
+
+## What's New
+* [Stateless Microservices]({% post_url 2018-09-06-stateless-microservices %})
+  * Deploy a regular microservice and manage them just like functions
+* [Growing list of end users](https://docs.openfaas.com/#users-of-openfaas)
+  * Shows growing traction as a production-ready FaaS platform
+  * Add your company [here](https://github.com/openfaas/faas/issues/776)
+* [OpenFaaS Cloud](https://docs.openfaas.com/openfaas-cloud/intro/)
+  * GitOps for your functions
+  * Becoming a platform to run OpenFaaS for multiple users or a team
```

Comment:
Feel like the post as a whole would be more in keeping with other posts with a prose based approach rather than the bullet list of points, wdyt?

(they would be super useful in the YT video description with timestamps for sure - always wished more people did that)

Reply:
I wasn't sure how to write up a recap without it being either a bullet list of points made, or just a transcript or the recording. 

My goal here was to provide enough information to satisfy the curious without making it a lot to digest. If someone wanted more information they could either view the video, or dive into the linked resource. 

I considered including links with the timestamps, but thought it didn't really fit in a blog post. As you said, that should be in the video description with the timestamp links (plus it's much easier to do there)